### ISSUE TITLE
Can't run on Debian 10

### ISSUE THREAD

**Issue by aufr33** [kkdai/youtube#124]
I downloaded the binary, but when I try to run, I get an error:

```
# /usr/bin/youtubedr
/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /usr/bin/youtubedr)
```

This is the last release of Debian and I have no way to update the glibc.

**Comment by kkc** [comment]
@aufr33  can you try `ldd --version` to see what version of GLIBC you currently use?
I am thinking we might need to use go 1.13 or go 1.12 to build youtubedr binary.

**Comment by aufr33** [comment]
ldd (Debian GLIBC 2.28-10) 2.28

**Comment by corny** [comment]
solved by f10c54c5e122c1fba390ca72da949d805fd4f3fe

**Comment by kkc** [comment]
cool thx a lot.
